metadata-ingestion/src/datahub/ingestion/source/grafana/models.py

```diff
@@ -0,0 +1,86 @@
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel, Field
+
+from datahub.emitter.mcp_builder import ContainerKey
+
```

Comment:
is there any public link with the grafana api models that we could add here as reference?